diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -167,7 +167,9 @@ dtb-$(CONFIG_ROCKCHIP_RK3399) += \
 	rk3399-rock960.dtb \
 	rk3399-rockpro64.dtb \
 	rk3399-tinker-2.dtb \
-	rk3399pro-rock-pi-n10.dtb
+	rk3399pro-rock-pi-n10.dtb \
+	rk3399-rp-king3399.dtb \
+	rk3399-zcube1-max.dtb
 
 dtb-$(CONFIG_ROCKCHIP_RK3568) += \
 	rk3568-evb.dtb
